internal/pkg/template/env.go

```diff
@@ -118,6 +119,12 @@ type VPCConfig struct {
 	Managed  ManagedVPC
 }
 
+// SecurityGroupConfig holds the fields to import security group config
+type SecurityGroupConfig struct {
+	Ingress string
```

Comment:
So does the marshaling logic depend on the fact that customers have to specify the exact same keys in the manifest vs in CF? 

For example, the manifest config 
```yaml
network:
  vpc:
    security_group:
      ingress:
        - IpProtocol: tcp
          FromPort: 0
          ToPort: 65535
        - IpProtocol: tcp
          FromPort: 1
          ToPort: 6
      egress:
        - IpProtocol: tcp
          FromPort: 0
          ToPort: 65535`
```
maps to the CloudFormation:
```yaml
      SecurityGroupIngress:
        - IpProtocol: tcp
          FromPort: 0
          ToPort: 65535
        - IpProtocol: tcp
          FromPort: 1
          ToPort: 6
      SecurityGroupEgress:
        - IpProtocol: tcp
          FromPort: 0
          ToPort: 65535
```

but this means that the customer has to write manifest keys in two separate capitalization styles, which may be confusing. Are there any other places in the manifest that we directly use CF keys versus remapping them to their snake case counterparts (`protocol`, `from_port`, and `to_port`)?